Watt this: printer_mod: in situ carriage block lubrication

jpeterse opened the pull request:
The purpose of the mod is to allow for lubrication of the printers linear rail carriage guide blocks while still installed on the printer. This is to make printer maintenance easier to perform.

<!--
Thank you for your interest in contributing to the VoronUsers repository, it is
highly appreciated!

**Please make sure the submission conforms to the rules outlined below. PRs which fail
to conform to the rules below are likely to be rejected.**
-->

  * [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folder and file naming:
    * Folders and filenames shouldn't contain spaces. Only letters `a-zA-Z`, numbers `0-9`, underscores `_`, hyphens `-` and periods `.`
    * Primary color: `part_xyz.stl`
    * Accent color: `[a]_part_xyz.stl`
    * Opaque color (Blocks light): `[o]_part_xyz.stl`
    * Clear/transparent color (Allows light): `[c]_part_xyz.stl`
    * Quantity, if more than one is needed: `part_xyz_x4.stl`
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submission includes a CAD file in the form of a `.STEP` or `.SCAD` file
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

<!--
Describe the submission further using the template provided below. The more 
details the better!
-->

#### Which mods/configurations/profiles are added by this PR?
Added Printer Mod watt-this/in-situ-lubrication

#### How was it tested? 
The modification was performed on my 2.4R2

#### Any background context you want to provide?
I found that with the increased printer speeds, specifically of the speeds that Voron printers can reach, the recommended travel distance between re-lubrication of the guide blocks come up quicker than I first thought. So I made this mod to make the process of lubricating the blocks easier.

#### Screenshots (if appropriate)
Pictures of the mod are added to the mod description

#### Further notes
I do hope the Voron team or Voron kit manufacturer will pick up on the idea, and make maintenance of the printers easier.

MapleLeafMakers commented:
I think it would be best to convert the images to png or jpeg.  There are no other .webp images in the repository and I'm not sure I want to be the first one to 'okay' them.  While browser-support seems to be good, other tools that interact with the repository (such as mods.vorondesign.com) might not handle them as well.

jpeterse commented on printer_mods/watt-this/in-situ-lubrication/README.md:
```diff
@@ -0,0 +1,232 @@
+         </td>
+      </tr>
+   </tbody>
+</table>
```
I was trying to make the readme more informative by adding image in the introduction without breaking the flow of the information. It was one of the comments I got on Discord, that it was difficult to understand how the mod worked. I couldn't get the layout to work standard markdown tables. Reading through this, did the instruction otherwise from through clear enough to communicate how the mod work and how to implement it?

jpeterse commented:
> I think it would be best to convert the images to png or jpeg. There are no other .webp images in the repository and I'm not sure I want to be the first one to 'okay' them. While browser-support seems to be good, other tools that interact with the repository (such as mods.vorondesign.com) might not handle them as well.

ok. I deliberately saved the images in webp to use a format specifically intended for the web. :-)
I'll convert the files and update the readme.

jpeterse commented:
Thank you for the comments and corrections. I have updated the Readme and converted the images.
I hope the updates make sense, and that the instructions for the mod are clear and easy to follow.

FHeilmann commented:
Hi, 

I want to thank you for your submission and want to personally apologize that your submission has not gotten the review it deserves.
We are in the process to restructure the repo to streamline reviews and hopefully solve this issue in the future.

Unfortunately, this means that we will require you to resubmit the mod under the new system.

Please check [here](https://github.com/VoronDesign/VoronUsers/wiki/VoronUsers-Overhaul) for details and instructions on how to proceed. 
If you have any questions, please reach out in the #voronuser_mods channel on the VoronDesign discord.
If you have since moved on from this mod and don't want to resubmit, no further action is necessary from your side.

Bests,

Florian a.k.a. Haribro
